TerminalDocs/customize-settings/key-bindings.md

```diff
@@ -0,0 +1,458 @@
+---
+title: Windows Terminal Key Bindings
+description: Learn how to create custom key bindings for Windows Terminal.
+author: cinnamon-msft
+ms.author: cinnamon
+ms.date: 05/19/2020
+ms.topic: overview
+ms.service: terminal
+---
+
+# Custom key bindings in Windows Terminal
+
+You can create custom key bindings inside the Windows Terminal that give you control of how you interact with the Terminal using your keyboard.
+
+## Key binding properties
+
+Key bindings can be constructed using the following properties.
+
+### Command
+
+The command executed when the associated keys are pressed.
+
+**Property name:** `command`
+
+**Necessity:** Required
+
+**Accepts:** String
+
+### Keys
+
+Defines the key combinations used to call the command.
+
+**Property name:** `keys`
+
+**Necessity:** Required
+
+**Accepts:** String or array[string]
+
+### Action
+
+Adds additional functionality to certain commands.
+
+**Property name:** `action`
+
+**Necessity:** Optional
+
+**Accepts:** String
+
+<br />
```

Comment:
Maybe you should provide an overview of how when to use "action" and when to use "command". Because it's one or the other. And this doesn't make it clear what the format looks like.

An example might be able to solve this problem though

Reply:
I reordered the sections so the key binding format is directly under this. I think that should help provide clarification?